src/commands/uplink/GetArchiveDaysMCResponse.ts

```diff
@@ -29,24 +29,23 @@ const COMMAND_ID = 0x1b;
 const COMMAND_BODY_MAX_SIZE = 5104;
 
 const examples: TCommandExampleList = [
-    // todo: fix
-    // {
-    //     name: '1 channel at ???',
-    //     parameters: {
-    //         channelList: [
-    //             {
-    //                 index: 0,
-    //                 days: [
-    //                     {value: 234, day: 0},
-    //                     {value: 332, day: 1}
-    //                 ]
-    //             }
-    //         ],
-    //         date: '2001-03-10T12:00:00.000Z',
-    //         days: 2
-    //     },
-    //     hex: {header: '1b 04', body: '2f 97 0c 01 00 00'}
-    // }
+    {
+        name: 'get day values from 2001.03.10 12:00:00 GMT for 0 channel',
+        parameters: {
+            startTime: 2678227200,
+            days: 2,
+            channelList: [
+                {
+                    index: 0,
+                    dayList: [
```

Comment:
let's write it in one line

and for constructor

Reply:
fixed